apps/api-extractor/src/collector/CollectorEntity.ts

```diff
@@ -175,6 +175,19 @@ export class CollectorEntity {
     return false;
   }
 
+  /**
+   * Return the first consumable parent that exports this entity. If there is none, returns
+   * `undefined`.
+   */
+  public getFirstExportingConsumableParent(): CollectorEntity | undefined {
```

Comment:
@fwienber I've converted this property to be a method because it does nontrivial work (iterating over a map).  

For example, someone might be tempted to write code like this:

```ts
if (entity.firstExportingConsumableParent?.exported 
  || entity.firstExportingConsumableParent?.consumable) {
}
```

...whereas denoting it as a function call makes the performance concern more conspicuous:

```ts
if (entity.getFirstExportingConsumableParent()?.exported 
  || entity.getFirstExportingConsumableParent()?.consumable) {
}
```